x-pack/plugins/ml/public/alerting/anomaly_detection_alerts_table/register_alerts_table_configuration.tsx

```diff
@@ -119,6 +120,7 @@ export function registerAlertsTableConfiguration(
 
   const config: AlertsTableConfigurationRegistry = {
     id: ML_ALERTS_CONFIG_ID,
+    cases: { featureId: CASE_APP_ID, owner: [CASE_APP_ID] },
```

Comment:
As discussed, the link to the case from the alerts table inside ML is currently broken:

<img width="376" alt="image" src="https://github.com/elastic/kibana/assets/7405507/68dc41d2-a904-4b4c-be13-56d339b3ff10">